packages/core/admin/admin/src/pages/Settings/components/Tokens/FormHead.tsx

```diff
@@ -151,6 +158,31 @@ export const FormHead = <TToken extends Token | null>({
                 url={`${regenerateUrl}${token?.id ?? ''}`}
               />
             )}
+            {token?.id && toggleToken && (
+              <Tooltip
+                label={
+                  !canShowToken &&
+                  formatMessage({
+                    id: 'Settings.tokens.encryptionKeyMissing',
+                    defaultMessage:
+                      'In order to view the token, you need an encryption key in the admin configuration',
+                  })
```

Comment:
I wonder if here we should distinguish between the 2 cases:
- no encryption key present at all
- encryption key available but not one that can display this token

?

Reply:
I could say a "valid encryption key"

Reply:
the frontend can't tell really if exists or not, it just knows "i don't have a key to display"